### PR TITLE
Feature/lp 2805 2806 ct products new additional cost types

### PR DESCRIPTION
**Pull request by thomaslkp123**

### CODE REVIEW

File: Beis.LearningPlatform.Web/Views/Shared/ComparisonToolViews/_PartialCompareProducts.cshtml [hunk 1]
```diff
@@ -147,110 +147,161 @@
...
-                    <p class="mobile-product-name">@ComparisonToolFieldValidator.ParseStringField(Model.products[i].product_name)</p>
-                    <p>@Model.products[i].contractDiscount</p>
-                </dd>
-            }
-        </div>
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key">
+                        Contract discount
+                    </dt>
+                    @for (var i = 0; i < Model.products.Count; i++)
```
Comment (jdssunil) [inline Beis.LearningPlatform.Web/Views/Shared/ComparisonToolViews/_PartialCompareProducts.cshtml:224]: Guess you can use foreach here then using index based loop

File: Beis.LearningPlatform.Web/Views/Shared/ComparisonToolViews/_PartialCompareProducts.cshtml [hunk 2]
```diff
@@ -147,110 +147,161 @@
...
+                            <h2 class="govuk-heading-m beis-blue-text govuk-!-margin-top-8">Transaction fees</h2>
+                        </div>
+                    </div>
+                    @foreach(var transactionFeeDescription in Model.ComparisonTransactionFeeDescriptions)
+                    {
+                        <div class="govuk-summary-list__row">
+                            <dt class="govuk-summary-list__key">
+                                @transactionFeeDescription
+                            </dt>
+                            @for (var i = 0; i < Model.products.Count; i++)
```
Comment (jdssunil) [inline Beis.LearningPlatform.Web/Views/Shared/ComparisonToolViews/_PartialCompareProducts.cshtml:246]: Guess you can use foreach here then using index based loop

File: Beis.LearningPlatform.Web/Views/Shared/ComparisonToolViews/_PartialCompareProducts.cshtml [hunk 3]
```diff
@@ -147,110 +147,161 @@
...
+                }
 
 
-                <dd class="govuk-summary-list__product@(i+1) mobile-side-by-side">
-                    <p class="mobile-product-name">@ComparisonToolFieldValidator.ParseStringField(Model.products[i].product_name)</p>
-                    @{
-                        var actionName = Model.JavascriptEnabled ? ActionConstants.ProductDetails : ActionConstants.ProductDetailsNoJs;
+                <div class="govuk-summary-list__row">
+                    <dt class="govuk-summary-list__key"></dt>
+                    @for (var i = 0; i < Model.products.Count; i++)
```
Comment (jdssunil) [inline Beis.LearningPlatform.Web/Views/Shared/ComparisonToolViews/_PartialCompareProducts.cshtml:286]: Guess you can use foreach here then using index based loop

**Review by jdssunil — APPROVED** [review]

**Review by TasTasniem — DISMISSED** [review]

**Review by JamesChristianson — DISMISSED** [review]

**Comment by thomaslkp123** [comment]
Deferred

**Pull request closed without merging**